src/main/java/org/skriptlang/skript/lang/comparator/Comparators.java

```diff
@@ -132,17 +133,10 @@ public static <Type1, Type2> Relation compare(@Nullable Type1 first, @Nullable T
 	public static <Type1, Type2> Comparator<Type1, Type2> getComparator(Class<Type1> firstType, Class<Type2> secondType) {
 		if (Skript.isAcceptRegistrations())
 			throw new SkriptAPIException("Comparators cannot be retrieved until Skript has finished registrations.");
-
-		int hash = Objects.hash(firstType, secondType);
-
-		Comparator<Type1, Type2> comparator = (Comparator<Type1, Type2>) QUICK_ACCESS_COMPARATORS.get(hash);
-
-		if (comparator == null) {
-			comparator = getComparator_i(firstType, secondType);
-			QUICK_ACCESS_COMPARATORS.put(hash, comparator);
-		}
-
-		return comparator;
+		return (Comparator<Type1, Type2>) QUICK_ACCESS_COMPARATORS.computeIfAbsent(
```

Comment:
Okay this was my bad but I looked at the description of `computeIfAbsent` and it also computes it when the key is mapped to null, so you can't use it 😅